chef/cookbooks/keystone/recipes/server.rb

```diff
@@ -545,6 +544,18 @@
     tenant_name tenant
     action :add_tenant
   end
+
+  ruby_block "saving id for default #{tenant} tenant" do
+    block do
+      # note that we know because of what's at the end of this recipe that the
+      # .openrc has been created, and that the openstack cli is available
+      tenant_id = %x[source /root/.openrc && openstack project show openstack | awk '/id/  { print $4 }'].chomp
```

Comment:
How about storing that as part of the keystone_register ```add_tenant``` action? It should be possible to add that to the LWRP. Though that also doesn't sound really clean :/